[system test] [doc] kafka package (#10650)
strimzi/strimzi-kafka-operator: development-docs/systemtests/io.strimzi.systemtest.kafka.KafkaVersionsST.md

# KafkaVersionsST

> **Description:** Test checking basic functionality for each supported Kafka version. Ensures that Kafka functionality such as deployment, Topic Operator, User Operator, and message transmission via PLAIN and TLS listeners are working correctly.

Reviewer: ```suggestion
**Description:** Tests the basic functionality for each supported Kafka version, ensuring that deployment, Topic Operator, User Operator, and message transmission via PLAIN and TLS listeners work correctly.
```

**Before tests execution steps:**

| Step | Action | Result |
| - | - | - |

> | 1. | Deploy cluster operator with default installation | Cluster operator is deployed |

Reviewer: ```suggestion
| 1. | Deploy Cluster Operator with default installation | Cluster Operator is deployed |
```

**Labels:**

* [kafka](labels/kafka.md)

<hr style="border:1px solid">

## testKafkaWithVersion

> **Description:** Test checking basic functionality for each supported Kafka version. Ensures that Kafka functionality such as deployment, Topic Operator, User Operator, and message transmission via PLAIN and TLS listeners are working correctly.

Reviewer: ```suggestion
**Description:** Verifies the basic functionality for each supported Kafka version.
```
